<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteSettingsDialog</class>
 <widget class="QDialog" name="RouteSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RouteSettingsDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/L2tpIPsecVpn.qrc">
    <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
  </property>
  <layout class="QGridLayout" name="m_pDialogGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>9</number>
   </property>
   <item row="2" column="1">
    <widget class="QPushButton" name="m_pAddButton">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="m_pDeleteButton">
     <property name="text">
      <string>Delete ...</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3">
    <widget class="QTableView" name="m_pRoutesTableView">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="toolTip">
      <string>route packages with the destination addresses listed here through the PPP tunnel.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_pButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="m_pUseExplicitRoutesRadioButton">
     <property name="text">
      <string>Use following explicit routes to intranet locations (split tunneling)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="m_pUseDefaultGatewayRadioButton">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Based on the &lt;span style=&quot; font-weight:600;&quot;&gt;Use default gateway on remote network&lt;/span&gt; setting, one of the following occurs when the VPN connection is active: &lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;When the &lt;span style=&quot; font-weight:600;&quot;&gt;Use default gateway on remote network&lt;/span&gt; check box is cleared, Internet locations are reachable and intranet locations are not reachable, except for those matching the network ID of the Internet address class of the assigned IP address. &lt;/li&gt;
&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;When the&lt;span style=&quot; font-weight:600;&quot;&gt; Use default gateway on remote network&lt;/span&gt; check box is selected (the default setting), all intranet locations are reachable and Internet locations are not reachable, except for the address of the VPN server and locations available through other routes. &lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Use default gateway on the remote network</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pUseDefaultGatewayRadioButton</tabstop>
  <tabstop>m_pUseExplicitRoutesRadioButton</tabstop>
  <tabstop>m_pRoutesTableView</tabstop>
  <tabstop>m_pAddButton</tabstop>
  <tabstop>m_pDeleteButton</tabstop>
  <tabstop>m_pButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/L2tpIPsecVpn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>RouteSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>RouteSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseDefaultGatewayRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pAddButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseDefaultGatewayRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pDeleteButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseDefaultGatewayRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pRoutesTableView</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
